edx_proctoring/views.py

```diff
@@ -779,3 +780,30 @@ def get(self, request):
             user_id=request.data.get('user_id', None),
             course_id=request.data.get('course_id', None)
         ))
+
+
+class ProctoredExamAttemptReviewStatus(AuthenticatedAPIView):
+    """
+    Endpoint for updating exam attempt's review status to acknowledged.
+    edx_proctoring/v1/proctored_exam/attempt/(<attempt_id>)/review_status$
+
+    Supports:
+        HTTP PUT: Update the is_status_acknowledge flag
+    """
+    def put(self, request, attempt_id):     # pylint: disable=unused-argument
+        """
+        Update the is_status_acknowledge flag for the specific attempt
+        """
+        try:
+            update_exam_attempt(attempt_id, is_status_acknowledged=True)
```

Comment:
We should assert here that the logged in user actually is the rightful owner of the exam attempt, otherwise someone could write a script (using a valid login session_id) which just goes through and acknowledges all attempts that do not belong to them. There should be other API endpoints in this file which do something similar.